.github/workflows/test.yml

```diff
@@ -40,7 +40,6 @@ jobs:
           key: ${{ runner.os }}-go-${{ matrix.go }}-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go-${{ matrix.go }}-
-            ${{ runner.os }}-go-
```

Comment:
Removing this because it looks like when caches are partially cleared, there could be some weird failures when e.g. a Windows build with go1.20 takes the cache from go1.18